src/server/server.config.js

```diff
@@ -24,6 +24,12 @@ const conf = convict({
     default: version,
     env: 'VERSION'
   },
+  phase: {
+    doc: 'The application release/phase version',
+    format: Number,
+    default: -1,
```

Comment:
```suggestion
    default: 0,
```

0 is the default value on goddapp